packages/dds/matrix/src/test/memory/.mocharc.js

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Mocha configuration file to run memory-profiling tests
+ */
+
+ module.exports = {
+    exit: true,
+    fgrep: ["@Benchmark", "@MemoryUsage"],
+    "node-option": ["expose-gc", "gc-global", "unhandled-rejections=strict"], // without leading "--"
+    recursive: true,
+    reporter: "../../../node_modules/@fluid-tools/benchmark/dist/MochaMemoryTestReporter.js", // Lerna hoists the external dependency on @fluid-tools/benchmark to the root
```

Comment:
```suggestion
    reporter: "@fluid-tools/benchmark/dist/MochaMemoryTestReporter.js", // Lerna hoists the external dependency on @fluid-tools/benchmark to the root
```

This should work as the default value. The one with the relative path is necessary in certain scenarios running locally, e.g. when symlinking the benchmark tool, but otherwise should not be necessary.